mkdocs.yml

```diff
@@ -508,8 +509,8 @@ nav:
             - Email: '@github(dhis2/dhis2-docs, src/developer/web-api/email.md, master)'
             - Data Store: '@github(dhis2/dhis2-docs, src/developer/web-api/data-store.md, master)'
             - Org unit profile: '@github(dhis2/dhis2-docs, src/developer/web-api/organisation-unit-profile.md, master)'
-            - Apps: '@github(dhis2/dhis2-docs, src/developer/web-api/apps.md, master)'
-        Loading Apps: '@github(dhis2/dhis2-docs, src/developer/apps.md, master)'
+            - '@github(dhis2/app-management-app, docs/developer/index.yml,master)'
+        '@github(dhis2/app-management-app, docs/developer/index.yml,master)'
```

Comment:
what is the difference between this and the entry above?

Reply:
Ah, thanks. I missed that one of the original lines is for the `web-api/apps.md`.

Reply:
Fixed it in 17f7cc4